bot.py

```diff
@@ -11,23 +15,33 @@
 from cogs.logging import Logging
 from utils.start_time import save_start_time
 
-def run_discord_bot():
-    GUILD_ID = env.get_guild_id()
 
+def get_guild(bot_ins):
+    _bot_id = int(os.getenv('force_guild', '0'))
```

Comment:
Tbh, this flag should be an argument, but we're not using parseargs, can be removed

Reply:
Still not quite sure what this is for? What is the difference compared to `GUILD_ID`? 🤔

Reply:
Allows you to override the GUILD

Reply:
As I understand, `FORCE_GUILD_ID` effectively fully replaces `GUILD_ID` when set and otherwise doesn't change any functionality, so you can just change `GUILD_ID` if you're testing in a different guild? 🤔 Let me know if I misunderstood.

Reply:
Yeap, that makes easier the development in other guilds

Reply:
Yea I understand that, but why not just change `GUILD_ID` to your testing guild in your `.env`?

Reply:
That's why I was saying that, it should be on argparse, modifying .env file is not a good practice, I can remove it but is not handy

Reply:
Not sure why it's bad practice, the production `.env` is ideally different anyway, with separate tokens and stuff. That said if we have a flexible config system where we can specify both configuration and this environment stuff both in files and CLI, I'm all for it.